lib/Yield/Support/GraphLayout/SugiyamaStyle/InternalCompute.h

```diff
@@ -169,6 +173,10 @@ bool computeInternal(InternalGraph &Graph, const Configuration &Configuration) {
   if (Configuration.UseOrthogonalBends)
     Prerouted = routeBackwardsCorners(Graph, Ranks, Lanes, Margin, EdgeGap);
 
+  // Restore the original edge directions.
+  // The graph will no longer be a DAG.
+  restoreEdgeDirections(Graph);
+
   // To simplify routing, order the edges first. This consumes the prerouted
```

Comment:
We used to consume BOTH the graph and the "prerouted edges" data-structure.

We still consume the edges, but not the graph.

So technically, both are correct. Except it might indeed be confusing. How do you think we should reword this?

Reply:
Just use the exact wording you've used here in this answer, an put it in the commit message. In this way even for the non-attentive reader, like I've been in this case, things are immediately obvious.

Reply:
Done.